<?xml version="1.0" encoding="UTF-8"?>

<ui version="4.0">
    <class>OutputDialog</class>

    <widget class="QDialog" name="OutputDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>350</width>
                <height>150</height>
            </rect>
        </property>

        <property name="windowTitle">
            <string>Resultado</string>
        </property>

        <property name="modal">
            <bool>true</bool>
        </property>

        <layout class="QVBoxLayout" name="verticalLayout">

            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">

                    <item>
                        <widget class="QLabel" name="icon_label">
                            <property name="minimumSize">
                                <size>
                                    <width>64</width>
                                    <height>64</height>
                                </size>
                            </property>
                            <property name="maximumSize">
                                <size>
                                    <width>64</width>
                                    <height>64</height>
                                </size>
                            </property>
                            <property name="text">
                                <string>(Icono)</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignmentFlag::AlignCenter</set>
                            </property>
                        </widget>
                    </item>

                    <item>
                        <widget class="QLabel" name="message_label">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Este es el mensaje de resultado que se mostrar√° al usuario.</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
                            </property>
                            <property name="wordWrap">
                                <bool>true</bool>
                            </property>
                        </widget>
                    </item>

                </layout>
            </item>

            <item>
                <widget class="QDialogButtonBox" name="button_box">
                    <property name="orientation">
                        <enum>Qt::Orientation::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::StandardButton::Ok</set>
                    </property>
                    <property name="centerButtons">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>

        </layout>
    </widget>

    <resources/>

    <connections>
        <connection>
            <sender>button_box</sender>
            <signal>accepted()</signal>
            <receiver>OutputDialog</receiver>
            <slot>accept()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>248</x>
                    <y>254</y>
                </hint>
                <hint type="destinationlabel">
                    <x>157</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>

        <connection>
            <sender>button_box</sender>
            <signal>rejected()</signal>
            <receiver>OutputDialog</receiver>
            <slot>reject()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>316</x>
                    <y>260</y>
                </hint>
                <hint type="destinationlabel">
                    <x>286</x>
                    <y>274</y>
                </hint>
            </hints>
        </connection>
    </connections>
</ui>